profile.template

```diff
@@ -143,7 +141,28 @@
               </div>
               {{/if}}
 
-              {{#if profile.certifications }}
+              {{#if profile.conferences}}
+              <div class="row">
+                <div class="col-md-12">
```

Comment:
Is this a responsive design?

Reply:
It's only as responsive as the Education and Training section was before. I haven't actually looked at the page structure properly to make any holistic design choices.